enterprise/backend/test/metabase_enterprise/sandbox/api/card_test.clj

```diff
@@ -66,11 +66,22 @@
                 :table_id        (mt/id :venues)}]]
 
         (testing "when getting values"
-          (is (=? {:values          ["African" "American" "Artisan"]
-                   :has_more_values false}
-                  (mt/user-http-request :rasta :get 200 (api.card-test/param-values-url card-id "abc")))))
+          (let [get-values (fn [user]
```

Comment:
adding a test here to make it clearer that we return more values if it's not sandboxed.